include/SFML/Window/VideoMode.hpp

```diff
@@ -53,12 +54,11 @@ class SFML_WINDOW_API VideoMode
     ////////////////////////////////////////////////////////////
     /// \brief Construct the video mode with its attributes
     ///
-    /// \param modeWidth        Width in pixels
-    /// \param modeHeight       Height in pixels
+    /// \param modeSize         Width and height in pixels
     /// \param modeBitsPerPixel Pixel depths in bits per pixel
     ///
     ////////////////////////////////////////////////////////////
-    VideoMode(unsigned int modeWidth, unsigned int modeHeight, unsigned int modeBitsPerPixel = 32);
+    VideoMode(const Vector2u& modeSize, unsigned int modeBitsPerPixel = 32);
```

Comment:
Note that this is now allowed:
```c++
sf::VideoMode mode = sf::Vector2u(800, 600);
```
Should this constructor be `explicit`?

Reply:
Good catch -- this also means that we will have silent code breaking from SFML 2 -> 3 (code still compiles, but causes nasty bugs at runtime).

Reply:
Okay! I just fixed it.